modules/virt-installing-virtctl-rhel8-rpm.adoc

```diff
@@ -3,10 +3,10 @@
 // * virt/getting_started/virt-using-the-cli-tools.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="virt-installing-virtctl-client-yum_{context}"]
-= Installing the virtctl RPM on {op-system-base} 8
+[id="virt-installing-virtctl-rhel8-rpm_{context}"]
+= Installing the virtctl RPM package on {op-system-base} 8
 
-You can install the `virtctl` RPM package on {op-system-base-full} 8 by enabling the {VirtProductName} repository and installing the `kubevirt-virtctl` package.
+You can install the `virtctl` RPM package on {op-system-base-full} 8 by enabling the {VirtProductName} repository and then installing `kubevirt-virtctl`.
```

Comment:
To be consistent with [this](https://github.com/openshift/openshift-docs/pull/67460/files#diff-c6a94af1adf62436f83e52a6e38c242ab796ff4b12e34a53b4c6c908a7ab2adeR19). 
```suggestion
You can install the `virtctl` RPM package on {op-system-base-full} 8 by enabling the {VirtProductName} repository and then installing the `kubevirt-virtctl` RPM package.
```